udata_front/theme/gouvfr/datagouv-components/src/components/ResourceAccordion/ResourceAccordion.vue

```diff
@@ -281,6 +295,7 @@ const owner = useOwnerName(communityResource.value);
 const lastUpdate = props.resource.last_modified;
 const availabilityChecked = props.resource.extras && 'check:available' in props.resource.extras;
 const unavailable = availabilityChecked && props.resource.extras['check:available'] === false;
+const downloadButtonTitle = unavailable ? t(`The {certifier} robot failed to access this file - Download file as {format}`, { certifier: config.title ,format }) : t(`Download file as {format}`, { format })
```

Comment:
I think `format` shows as Object here

Reply:
Oops, I moved it too fast out of the template